bob-apps/bin/blob_recovery_tool/command.rs

```diff
@@ -233,6 +245,9 @@ impl ValidateIndexCommand {
     }
 
     fn subcommand<'a, 'b>() -> App<'a, 'b> {
+        lazy_static::lazy_static! {
+                    static ref KEY_SIZE_HELP: String = format!("key size, supported 1, 2, 4, 8, 16, 32. {} used by default", PearlKey::LEN);
```

Comment:
Why is `lazy_static` used? The only place of access to this this field is in the next statement, so it looks like this message can be placed directly as an argument of `.help()`

Reply:
@vovac12

Reply:
Because App require `'static` lifetime

Reply:
Thanks for the information